src/components/ParametersMenu.tsx

```diff
@@ -68,6 +67,7 @@ function ParametersMenu(props: ParametersMenuProps) {
                                 className="inverse stepper cell-repetitions"
                                 id={`repetitionsAlongLatticeVector${label}`}
                                 value={
+                                    // @ts-ignore
                                     (viewerSettings as { [key: string]: any })[
```

Comment:
Can we use ViewerSettings type above?

Reply:
@zoomchik with help from Alex I went with defining type of key:
```
type ViewerSettingsKey = keyof ViewerSettings;
const key = `repetitionsAlongLatticeVector${label}` as ViewerSettingsKey;
```